Test/Install-WhiskeyNode.Tests.ps1

```diff
@@ -154,12 +167,102 @@ function WhenInstallingTool
     }
 }
 
+function Lock-File {
+    param(
+        $Seconds,
+        $File,
+        $DirName
+    )
+    Start-Job -ScriptBlock {    
+
+        while(-not (Test-Path -Path (Join-Path -Path $using:testRoot -ChildPath $using:DirName )))
+        {
+            continue;
+        }
+
+        $file = [IO.File]::Open($using:File, 'Open', 'Write', 'None')
+
+        try
+        {
+            Start-Sleep -Seconds $using:Seconds
+        }
+
+        finally
+        {
+            $file.Close()
+        }
+        
+    }
+
+    # Wait for file to get locked
+    do
+    {
+        Start-Sleep -Milliseconds 100
+        Write-Debug -Message ('Waiting for hosts file to get locked.')
+    }
+    while( (Get-Content -Path $File -ErrorAction SilentlyContinue ) )
+
+    $Global:Error.Clear()
+}
+
+function GivenAntiVirusLockingFiles
+{
+    param(
+    [String]$NodeVersion,
+
+    [switch]$AtLatestVersion,
+
+    [int]$Seconds
+    )
+
+    if( $AtLatestVersion )
+    {
+        $expectedVersion = Invoke-RestMethod -Uri 'https://nodejs.org/dist/index.json' |
+                                ForEach-Object { $_ } |
+                                Where-Object { $_.lts } |
+                                Select-Object -First 1
+        $NodeVersion = $expectedVersion.version
+    }
+
+    if( $IsWindows )
+    {
+        $platformID = 'win'
+        $extension = 'zip'
+    }
+    elseif( $IsLinux )
+    {
+        $platformID = 'linux'
+        $extension = 'tar.xz'
+    }
+    elseif( $IsMacOS )
+    {
+        $platformID = 'darwin'
+        $extension = 'tar.gz'
+    }
+
+    $extractedDirName = 'node-{0}-{1}-x64' -f $NodeVersion,$platformID
+    
+    $targetFilePath = Join-Path -Path $testRoot -ChildPath $extractedDirName
+    $targetFile = Join-Path -Path $targetFilePath -ChildPath 'LICENSE'
+
+    $job = Lock-File -Seconds $Seconds -File $targetFile -DirName $extractedDirName
+
+    try {

```

Comment:
All opening braces for language statements should be on their own line (opening braces for script blocks should not). Please update all opening braces to be on their own line.